.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: release
+
+on:
+  workflow_dispatch:
+    inputs:
+      semver:
+        description: The semver to use
+        required: true
+        default: patch
+        type: choice
+        options:
+          - patch
+          - minor
+          - major
+  pull_request:
+    types: [closed]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: nearform/optic-release-automation-action@v2
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN  }}
+          npm-token: ${{ secrets[format('NPM_TOKEN_{0}', github.actor)] || secrets.NPM_TOKEN }}
+          optic-token: ${{ secrets[format('OPTIC_TOKEN_{0}', github.actor)] || secrets.OPTIC_TOKEN }}
+          npm-tag: ${{ github.event.inputs.tag }}
+          semver: ${{ github.event.inputs.semver }}
+          commit-message: 'chore: release {version}'
+          build-command: npm ci
```

Comment:
I don't think we need this command. We usually need this when we need to build the package, e.g. transpile something. Plus besides being unnecessary this will break as npm ci needs a lockfile, which is not in this repo. Hence, remove